src/azure-cli/azure/cli/command_modules/synapse/tests/latest/assets/managedprivateendpoints.json

```diff
@@ -1,4 +1,5 @@
 {
-	"privateLinkResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test/providers/Microsoft.Storage/storageAccounts/test",
-	"groupId": "blob"
+	"privateLinkResourceId": "/subscriptions/813caa98-19eb-4c5e-b4d1-b962fc882993/resourceGroups/CreatePrivLinkService-rg/providers/Microsoft.Network/privateLinkServices/myPrivateLinkService",
```

Comment:
We'd better hide this user info by using a fake string.

Reply:
sure.